src/library/dlgcoverartfullsize.cpp

```diff
@@ -46,10 +46,27 @@ DlgCoverArtFullSize::DlgCoverArtFullSize(QWidget* parent, BaseTrackPlayer* pPlay
     setupUi(this);
 }
 
+void DlgCoverArtFullSize::closeEvent(QCloseEvent* event) {
+    hide();
```

Comment:
Why are the first two lines executed unconditionally?

It works as intended and the redundancy doesn't really matter. I am just asking questions that others might have in the future.

Reply:
We are not using the case without a parent in the current code base. It is just there in case we will use it, it does the right thing. Currently there is a VERIFY_OR_DEBUG_ASSERT for the parentless case in init anyway.